config.json

```diff
@@ -618,7 +618,38 @@
       "name": "Type Conversion"
     }
   ],
-  "key_features": [],
+  "key_features": [
+    {
+      icon: "web",
+      title: "Popular for Web and Mobile Development",
+      content: "Dart has the Flutter library, which can be used to quickly build web and smartphone applications."
```

Comment:
Web development can be a key feature as many libraries in Dart facilitate or support that. I would suggest dropping mention of Flutter here and instead use it as an example of the cross-platform feature.

If this section is for the web, then the language needs to focus on the web. Otherwise, this and the cross-platform sections are just repeating themselves.

Reply:
As an option, we can just collapse web and cross-platform into one section since web is part of cross-platform.

Reply:
I think combining web and cross-platform is fine, since all that is bundled together. In responses to your first comment, a single mention of Flutter in this section would probably be sufficient, avoiding mention of the library anywhere else.